src/main/java/at/hannibal2/skyhanni/config/features/event/hoppity/HoppityEggsConfig.java

```diff
@@ -218,8 +218,8 @@ public String toString() {
     public boolean petWarning = false;
 
     @Expose
-    @ConfigOption(name = "Prevent Missing Fish the Rabbit", desc = "Prevent closing a Meal Egg's inventory if Fish the Rabbit is present.")
+    @ConfigOption(name = "Prevent Missing Rabbit the Fish", desc = "Prevent closing a Meal Egg's inventory if Rabbit the Fish is present.")
     @ConfigEditorBoolean
     @FeatureToggle
-    public boolean preventMissingFish = true;
+    public boolean preventMissingRabbitTheFish = true;
```

Comment:
this will reset the config, please use a repo fix event to migrate the config. "why is the feature enabled again, i have just disabled it in the last beta"